File: src/library/scala/runtime/java8/JFunction0$mcD$sp.java

/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */

package scala.runtime.java8;

@FunctionalInterface
public interface JFunction0$mcD$sp extends scala.Function0, java.io.Serializable {
    double apply$mcD$sp();

    default Object apply() { return scala.runtime.BoxesRunTime.boxToDouble(apply$mcD$sp()); }
}

File: src/library/scala/runtime/java8/JFunction0$mcD$sp.scala
/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */
package scala.runtime.java8

@FunctionalInterface trait JFunction0$mcD$sp extends Function0[Any] with Serializable {
  def apply$mcD$sp: Double
  override def apply: Any = scala.runtime.BoxesRunTime.boxToDouble(apply$mcD$sp)
}